README.md

```diff
@@ -45,6 +45,24 @@ app.start(port=8080)
 
 ```
 
+## ![Python Version Support](https://img.shields.io/badge/Support-Version%20%E2%89%A5%203.7-brightgreen)
```

Comment:
Thanks @Oluwaseun241 . Great work. Just one last suggestion. Can you move the badge to the top level with the other badges?

Reply:
Done